app/models/institution_builder.rb

```diff
@@ -514,18 +514,45 @@ def self.build_institution_programs(version_number)
         graduate,
         full_time_modifier,
         length_in_hours,
-        version
+        version,
+        school_locale,
+        provider_website,
+        provider_email_address,
+        phone_area_code,
+        phone_number,
+        student_vet_group,
+        student_vet_group_website,
+        vet_success_name,
+        vet_success_email,
+        vet_tec_program,
+        tuition_amount,
+        length_in_weeks
       )
       SELECT
-        facility_code,
+        a.facility_code,
         program_type,
         description,
         full_time_undergraduate,
         graduate,
         full_time_modifier,
         length,
-        ?
-      FROM programs
+        ?,
+        school_locale,
+        provider_website,
+        provider_email_address,
+        phone_area_code,
+        phone_number,
+        student_vet_group,
+        student_vet_group_website,
+        vet_success_name,
+        vet_success_email,
+        vet_tec_program,
+        tuition_amount,
+        length_in_weeks
+      FROM programs a INNER JOIN edu_programs b
+        ON a.facility_code = b.facility_code
```

Comment:
Can a `facility_code` occur more than once in the `program.csv`?
I see in your `edu_program.csv` fixture that an `edu_program.csv` upload can.
So, for example, if there are four rows in `program.csv` for `facility_code` 13000313 and there are four rows in `edu_program.csv` for the same `facility_code`, you're going to get 16 rows inserted into `institution_programs` with all the various combinations.
Is what you want?

Reply:
Yes, institutions can have multiple programs.  Because this is also joining on the program name (`programs.description` and `edu_programs.vet_tec_program`) you wouldn't get 16 in the case of your example.  Assuming that the 4 rows in each joined table had 4 unique names that matched, you'd get 4.

Reply:
Ah yup.  Somehow I missed that.